src/Microsoft.DocAsCode.Build.Engine/HostService.cs

```diff
@@ -477,13 +477,18 @@ public void SaveIntermediateModel(IncrementalBuildContext incrementalContext)
                         }
 
                         // use copy rather than move because if the build failed, the intermediate files of last successful build shouldn't be corrupted.
-                        File.Copy(Path.Combine(incrementalContext.LastBaseDir, lfn), Path.Combine(incrementalContext.BaseDir, fileName));
+                        File.Copy(
+                            Path.Combine(Environment.ExpandEnvironmentVariables(incrementalContext.LastBaseDir), lfn),
```

Comment:
UT will be fail.

---
In reply to: [102642132](https://github.com/dotnet/docfx/pull/1339#discussion_r102642132) [](ancestors = 102642132)